packages/codemod/src/utils/prop-utils.ts

```diff
@@ -123,6 +150,29 @@ export function migratePropsNames(
   });
 }
 
+export function updatePropValues(
```

Comment:
```suggestion
export function updatePropsValue(
```
or
```suggestion
export function updatePropsValues(
```

Reply:
The function updates one prop and many value, so not sure I understand why it should be changed?

Reply:
oh, you are right. so same as for the static prop update comment - it shouldn't be a Record of a string to another Record, but rather to accept those inlined or in another structure. as the Record would always have one key and one value, right?

Reply:
The `valuesMapping` record can receive many values, the propName is a separate param, so that it has the option of updating several values on one prop.

Reply:
if we're talking on the same prop it can only have one value at a time, so there would always by one key
or am I missing something? can you give an example?

Reply:
Sure. For example I have a prop called size, and I want to update both the small and the medium values to be large.
I would call it like this:
`    updatePropValues(j, elementPath, "size", {
      small: {
        value: "large",
        type: j.Literal
      },
      medium: {
        value: "large",
        type: j.Literal
      }
    });`

Reply:
Ohhh ok I got you
I wonder if this should be in a separate calls to `updatePropValues`
but its cool for now, let's discuss it when we encounter such use case in the future

thanks for clarifying!!

Reply:
I assumed it would be more efficient in a case like this, so that you get the prop only once and then check if its current value matches any of the values that need to change. As supposed to checking it multiple times. Right now we don't have such a case, so do you think it will be better to just support one value and change it in the future if needed?

Reply:
I'm not sure actually
We can further discuss it when such a need raises, for now I wouldn't change it and remove an ability